src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -88,4 +88,34 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers
 
     response should be(None)
   }
+
+  "consignmentHasFiles" should "return true when files already associated with provided consignment id" in {
+    val fixedUuidSource = new FixedUUIDSource()
+    val fileUuid1 = UUID.randomUUID()
+    val fileUuid2 = UUID.randomUUID()
+    val mockResponse: Future[Boolean] = Future.successful(true)
+    val consignmentRepoMock = mock[ConsignmentRepository]
+    when(consignmentRepoMock.consignmentHasFiles(any[UUID])).thenReturn(mockResponse)
+
+    val consignmentService: ConsignmentService = new ConsignmentService(consignmentRepoMock, FixedTimeSource, fixedUuidSource)
+    val response: Boolean = consignmentService.consignmentHasFiles(UUID.randomUUID()).await()
+    verify(consignmentRepoMock, times(1)).consignmentHasFiles(any[UUID])
```

Comment:
This check is unnecessary - as a rule of thumb, you only need to verify that a mock is called if it has side-effects (such as modifying data, or sending an email or something).

In this case, what we really care about is that given the test setup, the response is correct, which you're checking for below.

Reply:
check removed